suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -603,6 +603,7 @@ export const getAccountSpecific = (
     if (networkType === 'ripple') {
         return {
             networkType,
+            // todo: shouldn't we move these fallbacks to blockchain-link?
```

Comment:
personally I don't like moving it there. I would like to move all the suite-side transformation away from blockchain-link and keep blockchain-link totally stupid, only fetching type-safe data from backneds. transformations should be done elsewhere.